VideoWeb/VideoWeb/ClientApp/src/app/shared/select-media-devices/select-media-devices.component.scss

```diff
@@ -1,11 +1,12 @@
 @import '~govuk-frontend/govuk/base';
-
+$talbet-portrait-width: 48em;
+$talbet-landscape-width: 64em;
```

Comment:
Change prefix to tablet.

Although it's actually recommended to avoid using device names as breakpoints and go for something like small, medium large. So it might be worth calling these medium-breakpoint and large-breakpoint.